docs/development.md

```diff
@@ -55,7 +55,7 @@ additional modules.
 
 ## Testing
 
-Automated tests require `docker`, `docker-compose`, `curl`, `mc` and `md5sum` to be
+Automated tests require `docker`, `docker-compose`, `curl`, and `mc` (the minio [cli tool](https://github.com/penpyt/asdf-mc)) to be
```

Comment:
Sorry for writing in closed issue, but official Minio CLI is there https://github.com/minio/mc.